### ISSUE TITLE
Error trying to import reducer from redux-form'

### ISSUE THREAD

**Issue by lucasribeiiroo** [erikras/redux-form#4287]
**ERROR MESSAGE -> withRef is removed. To access the wrapped instance, use a ref on the connected component**
the app was running ok, importing the reducer from redux-form and added to combine reducers, this error appears and i can't find the solution in the web.

```
import React from 'react'
import ReactDOM from 'react-dom'
import { Provider } from 'react-redux'
import thunk from 'redux-thunk'
import { createStore, applyMiddleware, compose } from 'redux'

import App from './components/App'
import reducers from './reducers'

const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
const store = createStore(
  reducers,
  composeEnhancers(applyMiddleware(thunk))
)

ReactDOM.render(
  <Provider store={store}>
  <App />
  </Provider>,
  document.querySelector('#root')
)


import { combineReducers } from 'redux'
import authReducer from './authReducer'
import {reducer} from '../../node_modules/redux-form/lib/reducer'

export default combineReducers({
  auth: authReducer,
  form: reducer,
})

```

**Comment by JeffreyATW** [comment]
This happened because you used React-Redux 6, which Redux Form does not support. Downgrade to 5 and it should work.

https://github.com/reduxjs/react-redux/releases/tag/v6.0.0

That said, I'd love to see Redux Form support React-Redux 6.

**Comment by wesleysd1989** [comment]
React Redux requires React 16.4 or later.

how your version on of react? 
i pass my react 16.6.3 to "react": "16.4.2", and work fine here.

**Comment by JerryGreen** [comment]
Experienced too, but looks like update fixed the problem.

[Found here](https://github.com/erikras/redux-form/issues/4286#issuecomment-462687896) [that is fixed in 8.0.0](https://github.com/erikras/redux-form/releases/tag/v8.0.0)

However, this may be in several different places in your projects (different libs) - so be careful. `redux-form` looks fine. I suggest to close the issue

**Comment by gustavohenke** [comment]
Thanks for the heads up, @JerryGreen! 😃
